server/manager/src/test/java/org/apache/accumulo/manager/split/SplitterTest.java

```diff
@@ -65,23 +65,23 @@ public void testShouldInspect() {
     expect(tabletMeta2.getExtent()).andReturn(ke2).anyTimes();
     replay(tabletMeta2);
 
-    assertTrue(splitter.shouldInspect(tabletMeta1));
```

Comment:
The method name for this test has shouldInspect in it, could rename the test method.

Reply:
Addressed in cfced0b